pyproject.toml

```diff
@@ -44,6 +44,9 @@ tiled = "tiled.commandline.main:main"
 
 # This is the union of all optional dependencies.
 all = [
+    "adbc_driver_manager",
```

Comment:
This section is used when tiled is installed like `pip install "tiled[all]"`. These three should also be added to the section `server`, below, so that they are included when tiled is installed like `pip install "tiled[server]"  # server only`.